privacyidea/static/components/config/views/config.events.details.html

```diff
@@ -107,54 +119,96 @@
+                                    <select class="form-control"
+                                            ng-if="condition.type!=='multi' && condition.value"
+                                            ng-disabled="!conditionCheckBox[conditionname]"
+                                            ng-required="condition.required && condition.value"
+                                            ng-model="conds[conditionname]"
+                                            ng-options="value for value in condition.value">
+                                    </select>
+                                    <div isteven-multi-select
+                                         ng-if="condition.type==='multi'"
+                                         ng-disabled="!conditionCheckBox[conditionname]"
+                                         input-model="condition.value"
+                                         output-model="conds[conditionname]"
+                                         button-label="icon name"
+                                         item-label="icon name maker"
+                                         tick-property="ticked">
+                                    </div>
+                                    <textarea type="text" rows="6"
+                                              ng-disabled="!conditionCheckBox[conditionname]"
+                                              ng-required="condition.required && !condition.value"
+                                              ng-if="condition.type==='text' && !condition.value"
+                                              class="form-control"
+                                              ng-model="conds[conditionname]">
+                                    </textarea>
+                                    <input class="form-control"
+                                           ng-disabled="!conditionCheckBox[conditionname]"
+                                           ng-required="condition.required && !condition.value"
+                                           ng-model="conds[conditionname]"
+                                           ng-if="(condition.type==='str' && !condition.value)
                             || condition.type==='regexp'">
-                        <p class="help help-block"
-                           ng-disabled="!conditionCheckBox[conditionname]">
-                            {{ condition.desc }}
-                        </p>
-                    </td>
-                </tr>
-            </table>
-        </div>
+                                    <p class="help help-block"
+                                       ng-disabled="!conditionCheckBox[conditionname]">
+                                        {{ condition.desc }}
+                                    </p>
+                                </td>
+                            </tr>
+                        </table>
+                    </div>
+                </uib-accordion>
+                    </div>
```

Comment:
Is this the closing ``</div>``  of line 148?
Please fix the indentation and move it to the left.

Reply:
Checked it and just moved the indentation myself in a5ff3f52